web/viewer.html

```diff
@@ -311,9 +309,6 @@
                 <button id="download" class="toolbarButton hiddenMediumView" title="Download" tabindex="34" data-l10n-id="download">
                   <span data-l10n-id="download_label">Download</span>
                 </button>
-                <a href="#" id="viewBookmark" class="toolbarButton hiddenSmallView" title="Current view (copy or open in new window)" tabindex="35" data-l10n-id="bookmark">
-                  <span data-l10n-id="bookmark_label">Current View</span>
-                </a>
 
                 <div class="verticalToolbarSeparator hiddenSmallView"></div>
```

Comment:
Also, doesn't this need to be updated as follows to prevent a useless separator at narrow viewer-widths?
```suggestion
                <div class="verticalToolbarSeparator hiddenMediumView"></div>
```